front/app/hooks/useHomepageSettingsFeatureFlag.test.ts

```diff
@@ -0,0 +1,62 @@
+import { renderHook } from '@testing-library/react-hooks';
```

Comment:
Great! Can you also add a test for the case we have a section that doesn't need checking in appConfig?

Reply:
Can you clarify?

Reply:
Just clarifying so we're on the same page: a test for e.g. the top_info section (that just needs the enabled check in homepageSettings, not the allowed check in appConfiguration).

Reply:
Just pushed that